src/components/calcite-date-picker-month/calcite-date-picker-month.scss

```diff
@@ -1,39 +1,42 @@
 .calender {
-  padding-bottom: 4px;
+  @apply  mb-1;
 }
 
 .week-headers {
-  display: flex;
-  flex-direction: row;
-  border-top: 1px solid var(--calcite-ui-border-3);
-  padding: 0 4px;
+  @apply flex
+    flex-row
+    border-t
```

Comment:
you need to set `border: 0 solid;` before you can use these. 

![Screen Shot 2021-03-18 at 4 21 47 PM](https://user-images.githubusercontent.com/1031758/111710069-2103a380-8806-11eb-9764-a577bab76d26.png)

Reply:
daaaaaaaaang, that looks pretty good though, maybe we should keep it like this?

:D

Reply:
This will be fixed soon